plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -2284,7 +2291,14 @@ private void updateTableStatistics(
             Optional<Instant> maxFileModificationTime,
             Collection<ComputedStatistics> computedStatistics)
     {
-        Optional<ExtendedStatistics> oldStatistics = statisticsAccess.readExtendedStatistics(session, location);
+        boolean recalculateAllStatistics = analyzeHandle.map(AnalyzeHandle::getAnalyzeType)
```

Comment:
```suggestion
        boolean recalculateAllStatistics = analyzeHandle
                .map(AnalyzeHandle::getAnalyzeType)
                .map(analyzeType -> analyzeType == FULL_REFRESH)
                .orElse(false);
```

Reply:
I agree with map on a new line the rest is similar